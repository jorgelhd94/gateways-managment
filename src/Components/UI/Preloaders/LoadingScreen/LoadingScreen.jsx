import React from 'react';
import styles from './LoadingScreen.module.css';
import Image from 'next/image';

const LoadingScreen = () => {
  return (
    <div className={styles.Loader}>
      <div className="flex justify-center h-screen w-screen">
        <Image src="/assets/img/cube.svg" width={60} height={60} priority />
      </div>
    </div>
  );
};

export default LoadingScreen;
