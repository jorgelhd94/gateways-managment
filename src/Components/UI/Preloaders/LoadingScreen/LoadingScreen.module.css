.Loader {
  position: absolute;
  height: 100vh;
  width: 100%;
  background: white;
  z-index: 9999;
}

.Loader div {
    background-color: #f1f2f3;
}
