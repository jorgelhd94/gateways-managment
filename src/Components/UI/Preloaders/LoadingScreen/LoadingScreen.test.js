import React from 'react';
import renderer from 'react-test-renderer';
import LoadingScreen from './LoadingScreen';

test('LoadingScreen render correctly', () => {
  const component = renderer.create(<LoadingScreen />);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
