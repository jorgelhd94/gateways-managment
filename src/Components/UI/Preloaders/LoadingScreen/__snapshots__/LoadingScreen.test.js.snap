// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoadingScreen render correctly 1`] = `
<div
  className="Loader"
>
  <div
    className="flex justify-center h-screen w-screen"
  >
    <span
      style={
        Object {
          "background": "none",
          "border": 0,
          "boxSizing": "border-box",
          "display": "inline-block",
          "height": "initial",
          "margin": 0,
          "maxWidth": "100%",
          "opacity": 1,
          "overflow": "hidden",
          "padding": 0,
          "position": "relative",
          "width": "initial",
        }
      }
    >
      <span
        style={
          Object {
            "background": "none",
            "border": 0,
            "boxSizing": "border-box",
            "display": "block",
            "height": "initial",
            "margin": 0,
            "maxWidth": "100%",
            "opacity": 1,
            "padding": 0,
            "width": "initial",
          }
        }
      >
        <img
          alt=""
          aria-hidden={true}
          src="data:image/svg+xml,%3csvg%20xmlns=%27http://www.w3.org/2000/svg%27%20version=%271.1%27%20width=%2760%27%20height=%2760%27/%3e"
          style={
            Object {
              "background": "none",
              "border": 0,
              "display": "block",
              "height": "initial",
              "margin": 0,
              "maxWidth": "100%",
              "opacity": 1,
              "padding": 0,
              "width": "initial",
            }
          }
        />
      </span>
      <img
        data-nimg="intrinsic"
        decoding="async"
        src="/assets/img/cube.svg"
        srcSet="/assets/img/cube.svg 1x, /assets/img/cube.svg 2x"
        style={
          Object {
            "border": "none",
            "bottom": 0,
            "boxSizing": "border-box",
            "display": "block",
            "height": 0,
            "left": 0,
            "margin": "auto",
            "maxHeight": "100%",
            "maxWidth": "100%",
            "minHeight": "100%",
            "minWidth": "100%",
            "objectFit": undefined,
            "objectPosition": undefined,
            "padding": 0,
            "position": "absolute",
            "right": 0,
            "top": 0,
            "width": 0,
          }
        }
      />
    </span>
  </div>
</div>
`;
